core/native-bridge.ts

```diff
@@ -513,6 +513,18 @@ const initBridge = (w: any): void => {
             method.toLocaleUpperCase() === 'OPTIONS' ||
             method.toLocaleUpperCase() === 'TRACE'
           ) {
+            // a workaround for following android webview issue:
+            // https://issues.chromium.org/issues/40450316
+            // Sets the user-agent header to a custom value so that its not stripped
+            // on its way to the native layer
+            if (platform === 'android' && options?.headers) {
+              const userAgent = headers.get('User-Agent') || headers.get('user-agent');
+              if (userAgent) {
```

Comment:
```suggestion
              if (userAgent !== null) {
```
This suggestion I made was not correct, it has to check for null because it's false for empty strings, and we should allow empty strings since iOS allow them.

The java code should also remove the two `!userAgentValue.isEmpty()` checks as we should allow to use empty strings as iOS does.